server/src/main/java/org/opensearch/index/query/functionscore/ScoreFunctionBuilder.java

```diff
@@ -58,11 +60,17 @@ public ScoreFunctionBuilder() {}
      */
     public ScoreFunctionBuilder(StreamInput in) throws IOException {
         weight = checkWeight(in.readOptionalFloat());
+        if (in.getVersion().onOrAfter(Version.V_2_0_0)) {
```

Comment:
@reta does it mean that this PR will target next major (2.0.0) release of OpenSearch, right? 🤔

Reply:
@lrynek the way we structure the pull requests: `main` first, backport to `1.x` after, if there are no major issues / objections, it could land in 1.3.0 (since this is not a breaking change).

Reply:
@reta thanks for explanation 👍 // _nomen omen_ 😅